flink-filesystems/pom.xml

```diff
@@ -34,7 +34,7 @@ under the License.
 	<packaging>pom</packaging>
 
 	<properties>
-		<fs.hadoopshaded.version>3.3.4</fs.hadoopshaded.version>
+		<fs.hadoopshaded.version>3.3.6</fs.hadoopshaded.version>
```

Comment:
There are notice files in the metadata in the subprojects that reference this number - they should be updated as well.